docs/intro.md

```diff
@@ -8,23 +8,26 @@ description: Intro to Rollkit, a modular framework for rollups.
 
 Welcome to Rollkit docs. We’re happy you made it here!
 
-Our mission is to empower developers to quickly innovate and create entire new classes of rollups with minimal tradeoffs.
-
-We’re setting the bar high for developers’ flexibility and ability to customize rollups however they see fit.
-
 :::tip Tip
 If you're familiar with Rollkit, you may want to skip to the [tutorials section](../category/tutorials)
 :::
 
 ## What is Rollkit?
 
-Rollkit is a rollup framework that gives developers the freedom to deploy rollups throughout the modular stack with minimal tradeoffs, opening new possibilities for rapid experimentation and innovation.
+Rollkit is a rollup framework that gives developers the freedom to deploy rollups throughout the modular stack, opening new possibilities for rapid experimentation and innovation.
+
+The Rollkit framework features a modular node that can run rollups and expose an **[ABCI](https://github.com/informalsystems/tendermint/tree/main/abci)**-compatible client interface, which can be used as a substitute for Tendermint in any ABCI-compatible blockchain application.\
```

Comment:
```suggestion
The Rollkit framework features a modular node that can run rollups and expose an **[ABCI](https://github.com/informalsystems/tendermint/tree/main/abci)**-compatible client interface, which can be used as a substitute for Tendermint, or any other ABCI-consensus mechanism, in any ABCI-compatible blockchain application.\
```

Reply:
hm, should we make it clear that it could be something that doesn't have tendermint, but is ABCI-compatible?